showcase/app/templates/components/advanced-table.hbs

```diff
@@ -250,7 +250,7 @@
 
   <Shw::Text::H2>Sticky header and sticky column</Shw::Text::H2>
 
-  <div class="shw-component-advanced-table-fixed-dimensions-wrapper">
+  <div class="shw-component-advanced-table-fixed-width-wrapper">
```

Comment:
[issue] I think the new class name has not been updated in Sass

Reply:
[todo] also check the `shw-component-advanced-table-fixed-height-wrapper` class which is used in the template but there's no counterpart in Sass (which means you can remove the `<div class="shw-component-advanced-table-fixed-height-wrapper">` wrapper.